### PR TITLE
store: add more logs.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing @zhangjinpeng1987

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -143,12 +143,18 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
             let local_state = try!(protobuf::parse_from_bytes::<RegionLocalState>(value));
             if local_state.get_state() == PeerState::Tombstone {
+                info!("region {:?} is tombstone in store {}",
```
Comment (BusyJay) [inline src/raftstore/store/store.rs]: I think debug is enough.

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -256,10 +262,20 @@ impl<T: Transport, C: PdClient> Store<T, C> {
...
-                                            region_id,
-                                            msg.get_region_epoch(),
-                                            msg.get_to_peer().get_id()));
+            let peer = match Peer::replicate(self,
+                                             region_id,
+                                             msg.get_region_epoch(),
+                                             msg.get_to_peer().get_id()) {
+                Ok(peer) => peer,
+                Err(e) => {
+                    error!("from {:?} replicates to {:?} for {} failed {:?}",
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: s/replicates/peer/

Comment (ngaut) [inline src/raftstore/store/store.rs]: region id

**Comment by BusyJay** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**